### PR TITLE
doc/rbd: refine "Retrieving Image Information"

### PR DESCRIPTION
**Pull request by zdover23**
Refine the text and prompts in "Retrieving Image Information" in doc/rbd/rados-rbd-cmds.rst.

https://tracker.ceph.com/issues/57001

Signed-off-by: Zac Dover <zac.dover@gmail.com>

## Contribution Guidelines
- To sign and title your commits, please refer to [Submitting Patches to Ceph](https://github.com/ceph/ceph/blob/main/SubmittingPatches.rst).

- If you are submitting a fix for a stable branch (e.g. "pacific"), please refer to [Submitting Patches to Ceph - Backports](https://github.com/ceph/ceph/blob/master/SubmittingPatches-backports.rst) for the proper workflow.

## Checklist
- Tracker (select at least one)
  - [ ] References tracker ticket
  - [ ] Very recent bug; references commit where it was introduced
  - [ ] New feature (ticket optional)
  - [x] Doc update (no ticket needed)
  - [ ] Code cleanup (no ticket needed)
- Component impact
  - [ ] Affects [Dashboard](https://tracker.ceph.com/projects/dashboard/issues/new), opened tracker ticket
  - [ ] Affects [Orchestrator](https://tracker.ceph.com/projects/orchestrator/issues/new), opened tracker ticket
  - [x] No impact that needs to be tracked
- Documentation (select at least one)
  - [x] Updates relevant documentation
  - [ ] No doc update is appropriate
- Tests (select at least one)
  - [ ] Includes [unit test(s)](https://docs.ceph.com/en/latest/dev/developer_guide/tests-unit-tests/)
  - [ ] Includes [integration test(s)](https://docs.ceph.com/en/latest/dev/developer_guide/testing_integration_tests/)
  - [ ] Includes bug reproducer
  - [x] No tests

<details>
<summary>Show available Jenkins commands</summary>

- `jenkins retest this please`
- `jenkins test classic perf`
- `jenkins test crimson perf`
- `jenkins test signed`
- `jenkins test make check`
- `jenkins test make check arm64`
- `jenkins test submodules`
- `jenkins test dashboard`
- `jenkins test dashboard cephadm`
- `jenkins test api`
- `jenkins test docs`
- `jenkins render docs`
- `jenkins test ceph-volume all`
- `jenkins test ceph-volume tox`
- `jenkins test windows`
</details>

### CODE REVIEW

File: doc/rbd/rados-rbd-cmds.rst
Comment (anthonyeleven) [inline doc/rbd/rados-rbd-cmds.rst:140]: At a former employer we had the convention of naming RBD images with `userid/<uuid>`, which collided with this syntax in a rather annoying way.  I wonder if we might do well to advise instead (or at least additionally) e.g. `rbd -p {pool-name| info {image-name}`

**Review by anthonyeleven — APPROVED** [review]
One comment, non-blocking but please consider.

File: doc/rbd/rados-rbd-cmds.rst
Comment (zdover23) [reply doc/rbd/rados-rbd-cmds.rst:140]: It's a good point. I'll incorporate it.

**Comment by zdover23** [comment]
https://github.com/ceph/ceph/pull/49350 - Quincy backport
https://github.com/ceph/ceph/pull/49351 - Pacific backport

File: doc/rbd/rados-rbd-cmds.rst
Comment (idryomov) [inline doc/rbd/rados-rbd-cmds.rst:153]: Where did `|` came from?  It's going to be interpreted as a pipe to `info` by the shell...

Comment (idryomov) [reply doc/rbd/rados-rbd-cmds.rst:153]: Also, might have been useful to link "Image, snap, group and journal specs" section of the man page (https://docs.ceph.com/en/latest/man/8/rbd/#image-snap-group-and-journal-specs) here.

**Review by idryomov — COMMENTED** [review]
@zdover23 @anthonyeleven @colemitchell I want to raise two process-related issues with the on-going "doc refining" work:

- I don't think splitting these mostly minor changes into tiny, a couple dozen lines, PRs makes sense.  I would prefer to see a single PR per page (`.rst` file) so that it could be reviewed once and consistency across all sections could be easily enforced.

- This stuff is merged way too quickly.  `ceph/rbd` team is a code owner for `doc/rbd` so a review is auto-requested but PRs are merged without waiting for it to happen.  As it stands, this just spams people's inboxes -- even the backport PRs, let alone the original PR, are merged before anyone gets a chance to take a look.

**Comment by zdover23** [comment]
@idryomov, I hear you. I'll make a single PR next week for https://docs.ceph.com/en/latest/rbd/rbd-snapshot/, and I hope that the single PR will keep your inbox from being flooded with docs-related notifications that aren't helpful.

I apologize for opening these PRs and then merging them as quickly as we have been. In almost every case, each of the docs PRs that we open concerns an issue that has been under discussion for hours and the PR represents the final phase of the discussion and revision. I'll insist that we keep these RBD PRs open for however long you ask. Does three days seem like a good amount of time for technical review?

File: doc/rbd/rados-rbd-cmds.rst
Comment (zdover23) [reply doc/rbd/rados-rbd-cmds.rst:153]: > Where did `|` come from? It's going to be interpreted as a pipe to `info` by the shell...

It's true. It's true.  This is my fault. I'll remove it in a follow-up PR.

Comment (zdover23) [reply doc/rbd/rados-rbd-cmds.rst:153]: > Also, might have been useful to link "Image, snap, group and journal specs" section of the man page (https://docs.ceph.com/en/latest/man/8/rbd/#image-snap-group-and-journal-specs) here.

I'll link to this in a future PR. One of the reasons that we have developed this accidentally irritating workflow is so that we can ingest suggestions like this.

As I said (or at least meant to) in another comment on this page, I'm sorry that our docs workflow spammed your inbox and made your workweek crummier than it otherwise would have been.

**Comment by idryomov** [comment]
> I'll insist that we keep these RBD PRs open for however long you ask. Does three days seem like a good amount of time for technical review?

The proper workflow would be to keep them open until they are approved on behalf of `ceph/rbd` by a member of the RBD team.  If you don't a get response for few days, leave a ping comment in the PR or reach out directly.

**Pull request merged**
